docs/assistant/security-assistant.asciidoc

```diff
@@ -2,4 +2,97 @@
+
+[discrete]
+[[start-chatting]]
+== Start chatting
+
+To open the Security Assistant, press *Cmd + ;* (or *Ctrl + ;* in Windows) from anywhere in the {security-app}. This opens the *Welcome* conversation context, where you can ask general questions about {elastic-sec}.
+
+You can also chat with the Security Assistant from several areas of {elastic-sec}, and context-specific data and prompts will populate your conversation.
+
+* <<view-alert-details, Alert details>> or Event details flyout: Click *Chat* while viewing the details of an alert or event.
+* <<rules-ui-management, Rules page>>: Select one or more rules, then click the magic wand icon (🪄✨) at the top of the page next to the *Rules* title.
+* <<data-quality-dash, Data Quality dashboard>>: Select the *Incompatible fields* tab, then click *Chat*. (This is only available for fields marked red, indicating they're incompatible).
+* <<timelines-ui, Timeline>>: Select the *Security Assistant* tab.
+
+All chat history persists within your session, so you can navigate away to other areas of {elastic-sec}, then later return to ongoing conversations.
+
+[discrete]
+[[interact-with-assistant]]
+== Interact with the Security Assistant
+
+Use these features to adjust and act on your conversations with the Security Assistant:
+
+* Select a _system prompt_ at the beginning of a conversation to establish how detailed and technical you want the Security Assistant's answers to be.
++
+[role="screenshot"]
+image::images/system-prompt.gif[The system prompt drop-down menu,90%]
++
+NOTE: The system prompt is only configurable at the start of a conversation. To reconfigure, clear the chat and start a new conversation.
+
+* Select a _quick prompt_ at the bottom of the chat window to get help writing a prompt for a specific purpose, such as summarizing an alert or converting a query from a legacy SIEM to {elastic-sec}. Available quick prompts vary based on context. You can also add your own custom quick prompts for questions that you frequently ask the Security Assistant.
++
+[role="screenshot"]
+image::images/quick-prompts.png[Quick prompts highlighted below a conversation,90%]
+
+* Use available buttons to perform actions in the conversation history and prompt entry area:
+
+** *Add note to timeline* (image:images/icon-add-note.png[Add note icon,16,16]): Create a note in Timeline using the selected text.
+** *Add to existing case* (image:images/icon-add-to-case.png[Add to case icon,19,16]): Add a comment to an existing case using the selected text.
+** *Copy to clipboard* (image:images/icon-copy.png[Copy to clipboard icon,17,18]): Copy the text to clipboard to paste elsewhere. This is also helpful for resubmitting a previous prompt.
+** *Add to timeline* (image:images/icon-add-to-timeline.png[Copy to clipboard icon,17,18]): Add a filter or query to Timeline using the text. This button appears for certain queries in the Security Assistant's responses.
```

Comment:
Would it make sense to move this up, after the first bullet? In terms of organization, it might be good to group that functionality.

Reply:
Not exactly, because the user will encounter the two buttons (**Add note to timeline** and **Add to timeline**) in different parts of the app. This will be delineated more in the next iteration of docs, where we'll go into specific workflows in more depth, but for now this list is generally ordered by where the user will find the buttons.

Reply:
Makes sense - I figured this list might've been ordered to reflect the UI order.